google-cloud-workflows-v1/.repo-metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "api_id": "workflows.googleapis.com",
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-workflows-v1/latest",
+    "distribution_name": "google-cloud-workflows-v1",
+    "language": "ruby",
+    "name": "workflows",
+    "name_pretty": "Workflows V1 API",
+    "product_documentation": "https://cloud.google.com/workflows/",
+    "repo": "googleapis/google-cloud-ruby",
+    "requires_billing": true,
+    "ruby-cloud-description": "Workflows link series of serverless tasks together in an order you define. Combine the power of Google Cloud's APIs, serverless products like Cloud Functions and Cloud Run, and calls to external APIs to create flexible serverless applications. Workflows requires no infrastructure management and scales seamlessly with demand, including scaling down to zero.. Note that google-cloud-workflows-v1 is a version-specific client library. For most uses, we recommend installing the main client library google-cloud-workflows instead. See the readme for more details.",
```

Comment:
Extra period: `including scaling down to zero.. Note that`

Reply:
Whoops, good catch. Looks like it's present in the v1beta client as well. I'll need to fix both in the `BUILD.bazel` files upstream. For now, since this is a minor typo, I think we can proceed with initial generation with it.